src/org/apache/xerces/impl/xpath/regex/RegularExpression.java

```diff
@@ -27,7 +27,7 @@
  * A regular expression matching engine using Non-deterministic Finite Automaton (NFA).
  * This engine does not conform to the POSIX regular expression.
  *
- * <h3>How to use</h3>
+ * <h1>How to use</h1>
```

Comment:
h1 is typically used for the entire page; maybe start with h2
the warning was 

```error: heading used out of sequence: <H3>, compared to implicit preceding heading: <H1>
```
 so there's already an H1 here and we should be fine to use an h2